src/js/utils.js

```diff
@@ -16,6 +16,27 @@ exports.is_sample_genetically_altered = function is_sample_genetically_altered(d
   || datum.protein !== undefined;
 };
 
+exports.makeD3SVGElement = function(tag) {
```

Comment:
I think it's better to pass around the DOM object in this case just because we don't want to get too tied up in being a middleman and the renderer is specifically a d3 svg renderer so we don't need to allow any more generality than that.